### PR TITLE
Add continuous testing and multiple instance support to CI

### PR DESCRIPTION
**Pull request by everestmz**
cc: @tanin47

### CODE REVIEW

File: ci/continuous_integration/daemon/main.py [hunk 1]
```diff
@@ -129,31 +132,53 @@ def call_with_depot_tools(command, cwd=CHROMIUM_SRC):
...
+  if not os.path.exists(TOOL_SOURCE):
+    subprocess.check_call(('git clone https://github.com/google/clusterfuzz'
+                           '-tools.git'), shell=True, cwd=HOME)
+  subprocess.check_call('git pull', shell=True, cwd=TOOL_SOURCE)
+  subprocess.check_call('./pants binary tool:clusterfuzz-ci', shell=True,
+                        cwd=TOOL_SOURCE)
+  os.remove(BINARY_LOCATION)
+  shutil.copy(os.path.join(TOOL_SOURCE, 'dist', 'clusterfuzz-ci.pex'),
+                           BINARY_LOCATION)
+  global binary_sha
```
Comment (tanin47) [inline ci/continuous_integration/daemon/main.py]: I think it's better to return `binary_sha`.

File: ci/continuous_integration/daemon/main.py [hunk 2]
```diff
@@ -129,31 +132,53 @@ def call_with_depot_tools(command, cwd=CHROMIUM_SRC):
...
+  binary_sha = subprocess.check_output('git rev-parse HEAD', shell=True,
+                                       cwd=BINARY_LOCATION)
+
+
+def reset_and_run_testcase(testcase_id, test_type, release):
   """Resets the chromium repo and runs the testcase."""
 
   delete_if_exists(CHROMIUM_OUT)
   delete_if_exists(CLUSTERFUZZ_DIR)
+  if release == 'master':
```
Comment (tanin47) [inline ci/continuous_integration/daemon/main.py]: Set the version here.

Comment (everestmz) [inline ci/continuous_integration/daemon/main.py]: What do you mean set the version - like get the git sha?

Comment (tanin47) [inline ci/continuous_integration/daemon/main.py]: I mean rename the `binary_sha` to `version`. And make it a local variable... not a global one.

Comment (everestmz) [inline ci/continuous_integration/daemon/main.py]: Updated

**Pull request merged**